Simulation.java

```diff
@@ -66,11 +68,12 @@ public static void main(String[] args) {
 
         String graknHostUri = getOption(commandLine, "g").orElse(GraknClient.DEFAULT_URI);
         long seed = getOption(commandLine, "s").map(Long::parseLong).orElseGet(() -> {
-            long s = new Random().nextLong();
+            long s = 1;
             System.out.println("No seed supplied, using random seed: " + s);
             return s;
         });
-        int iterations = getOption(commandLine, "i").map(Integer::parseInt).orElse(10);
+        int numIterations = 10;
```

Comment:
Does it make more sense perhaps to make this a constant at the start of the file?